src/Tizen.NUI/src/public/Rendering/Renderer.cs

```diff
@@ -587,7 +594,7 @@ public Geometry GetGeometry()
             System.IntPtr cPtr = Interop.Renderer.GetGeometry(SwigCPtr);
             Geometry ret = this.GetInstanceSafely<Geometry>(cPtr);
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
-            return ret;
+            return CurrentGeometry = ret;
```

Comment:
Why don't you directly return `CurrentGeometry`?
Is it possible to update by Native layout without update of C# layer?